templates/base.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="icon" href="/static/images/favicon.ico" type="icon" />
     <link rel="stylesheet" href="/static/css/main.css" />
+    <script language="JavaScript">
+      function showInput() {
+          function letter() {
+            var randLetter = String.fromCharCode(97+Math.floor(Math.random() * 26))
+            return randLetter
+          }
+          function number() {
+            randNumber = Math.floor(Math.random() * 10)
+            return randNumber
+          }
+          input = document.getElementById("user_input").value;
+          console.log('input', input.toLowerCase().replace(/ /g, "-"))
```

Comment:
```suggestion
          console.log('input', input.toLowerCase().replace(/ /g, "-"));
```

Reply:
I would suggest that we should use the same regexp we used on the backend: https://github.com/Bag-of-Party/python-website/blob/e697ba283b0b3bd56e09edf742902aaadee337b0/bagofparty.py#L19